player.py

```diff
@@ -143,7 +144,7 @@ def __init__(
         self.rip_duration = 12 + 4 * rip_glyph + 4 * t7_2p
         self.rake_duration = 9 + 3 * t9_2p
         self.lacerate_multi = 1 + 0.05 * t7_2p
-        self.lacerate_dot_multi = 1 + 0.05 * t9_2p
+        self.lacerate_dot_multi = 1 + 0.05 * t9_2p + 0.2 * t10_2p
```

Comment:
This should be `(1 + 0.05 * t9_2p) * (1 + 0.2 * t10_2p)` I think, since the two bonuses should scale multiplicatively with one another.